src/vector.c

```diff
@@ -150,6 +168,13 @@ vector_in(PG_FUNCTION_ARGS)
 	char	   *stringEnd;
 	Vector	   *result;
 
+	/*
+	 * Note: we currently allow whitespace between, but not within,
+	 * dimension items.
```

Comment:
I'm not sure this comment makes sense for vectors, so let's remove it

Reply:
copy that